src/inscription.rs

```diff
@@ -36,6 +36,22 @@ impl Inscription {
     InscriptionParser::parse(&tx.input.get(0)?.witness).ok()
   }
 
+  pub fn from_transaction_experiment(tx: &Transaction) -> Vec<Option<Inscription>> {
+    let mut inscriptions = Vec::with_capacity(&tx.input.len());
```

Comment:
Unsure why this doesn't make the CI fail but I get an error here that it expects a `usize` and not a `&usize`. See also https://doc.rust-lang.org/std/vec/struct.Vec.html#method.with_capacity

Reply:
thanks!